recipes/cryptopp/all/patches/0001-fix-msvc-arm64-8.7.0.patch

```diff
@@ -0,0 +1,33 @@
+--- a/arm_simd.h
++++ b/arm_simd.h
+@@ -307,28 +307,30 @@ inline uint64x2_t PMULL_HIGH(const uint64x2_t a, const uint64x2_t b)
+ #endif
+ }
+ 
++#if !defined(_MSC_VER) || defined(_M_ARM)
```

Comment:
https://github.com/weidai11/cryptopp/commit/31fa3384160071793bc428a32383938551b3652c fully removes this function while you only disable it for MSVC and ARM. Is that intentional?

In any case, it's probably a good idea to link to the upstream commit from this patch for documentation purposes.